comps/retrievers/langchain/vdms/requirements.txt

```diff
@@ -0,0 +1,17 @@
+docarray[full]
+easyocr
+einops
+fastapi
+langchain-community
+langchain-core
+langchain-huggingface
+langsmith
```

Comment:
Please remove the langsmith related code

Reply:
done